content/courses/onchain-development/intro-to-anchor.md

```diff
@@ -40,25 +45,31 @@ with writing a Solana program.
 
 Anchor uses macros and traits to generate boilerplate Rust code for you. These
 provide a clear structure to your program so you can more easily reason about
-your code. The main high-level macros and attributes are:
+your code and focus more on the functionality of your program. The main
+high-level macros and attributes are:
```

Comment:
Again this isn't you, I just think the old version of this is really vague and we can do better if we provide a more specific example: 

> Anchor uses macros and traits to generate boilerplate Rust code for you. For example, Solana programs using Anchor look like regular Rust modules - they use the `mod` keyword, and have functions inside. By applying the `program` macro to that module though, that regular Rust module becomes a Solana program - it will have an address, send incoming instructions to the right instruction handler function, and more. Anchor will generate the code to make the module a Solana program for us at compile time - we don't have to see that code or think about that code, we just apply the `program` macro and the work is done for us. 

Just a suggestion. Feel free to play with it!